test/kdump_test.rb

```diff
@@ -729,4 +729,29 @@
       expect(memory).to eq(high: "64", low: "32")
     end
   end
+
+  describe ".AutoYaST" do
+    before do
+      Yast::Mode.SetMode(mode)
+    end
```

Comment:
We prefer to have things like `allow(Yast:Mode).to receive(:autoinst).and_return true` instead of `SetMode`. It's safer.